### PR TITLE
To Do list interactive list final :- module 2

### PR DESCRIPTION
**Pull request by yusufyusie**
### In this project, you will add some functionality to your application to make it interactive
**The updated tasks list will be stored in local storage  in the following ways**

- Added a new JavaScript file and import it as a module:
- Added event listener to the checkbox (change).
- Updated items object's value for completed key upon user actions.
- Implemented a function for the "Clear all completed" button (use filter() method).
- Stored the updated array of items in local storage, so the user gets the correct list of values after the page reloads (

### CODE REVIEW

File: src/modules/Completelist.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    clearButton.addEventListener('click', () => {
+      let todolists = LocalStorage.getToDoLists();
+      todolists = todolists.filter((elemnt) => elemnt.comp !== true);
+      localStorage.setItem('todolists', JSON.stringify(todolists));
+      window.location.reload();
+      LocalStorage.resetIndex();
+    });
+  }
+}
+
```
Comment (zieeco) [inline src/modules/Completelist.js]: - Hey, when users click the checkboxes, they don't get updated from true to false in the local storage and the clear all completed doesn't seem to clear any completed task. please, double-check this to make sure your app meets this part of the project requirement as much as you can.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @yusufyusie,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice implementing the functionality ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions

- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/Completelist.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ class Completelist {
     const clearButton = document.querySelector('.clear');
     clearButton.addEventListener('click', () => {
       let todolists = LocalStorage.getToDoLists();
-      todolists = todolists.filter((elemnt) => elemnt.comp !== true);
+      todolists = todolists.filter((elemnt) => elemnt.completed !== true);
       localStorage.setItem('todolists', JSON.stringify(todolists));
       window.location.reload();
```
Comment (TadesseDev) [inline src/modules/Completelist.js]: - [Optional] You did great adding the changes as requested @yusufyusie. But, I notice your task remains an input field after adding the task. It is not a good Ides, It should be turned into the input field usually when the user clicks on the edit task. for the time being, I would suggest adding the task as `<p>` or something else other than the input field.

**Review by TadesseDev — APPROVED** [review]
### Hi @yusufyusie 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**